source/_integrations/google_cloud.markdown

```diff
@@ -136,6 +140,10 @@ data:
       - wearable-class-device
 ```
 
+## Action say (legacy)
+
+The `tts.google_cloud_say` action can be used when configuring the legacy `google_cloud` text-to-speech platform in `configuration.yaml`. We recommend new users to instead set up the integration in the UI and use the `tts.speak` action with the corresponding Google Cloud text-to-speech entity as target. If you are an existing user of `tts.google_cloud_say`, you can still use it but don't remove the legacy `google_cloud` text-to-speech platform in `configuration.yaml`. If you remove it, you will have to manually migrate to `tts.speak`.
```

Comment:
I think we should rather mention this at the config part of the docs. The downside is that it's not deprecated so theoretically their way is still supported

Reply:
I'd rather have it here at the bottom since it only affects few old users. And I think it's more discoverable here in the `google_cloud_say` section. Few users read the documentation in general. They will mostly look at it whenever they will encounter an issue. If they manually remove the legacy TTS platform, their google_cloud_say calls will fail so they will search for that in the docs.

Reply:
Fair